src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -2814,63 +2814,6 @@ namespace Microsoft.FSharp.Core
             /// <returns>The three dimensional sub array from the given indices.</returns>
             val GetArraySlice3D : source:'T[,,] -> start1:int option -> finish1:int option -> start2:int option -> finish2:int option -> start3:int option -> finish3:int option -> 'T[,,]
 
-            /// <summary>Gets a 2D slice of a 3D array.</summary>
-            /// <param name="source">The source array.</param>
-            /// <param name="index1">The fixed index of the first dimension.</param>
-            /// <param name="start2">The start index of the second dimension.</param>
-            /// <param name="finish2">The end index of the second dimension.</param>            
-            /// <param name="start3">The start index of the third dimension.</param>
-            /// <param name="finish3">The end index of the third dimension.</param>
-            /// <returns>The two dimensional sub array from the given indices.</returns>
-            val inline GetArraySlice3DFixedSingle1 : source:'T[,,] ->  index1:int -> start2:int option -> finish2:int option -> start3:int option -> finish3:int option -> 'T[,]
```

Comment:
These would have to remain in the DLL to avoid a binary breaking change for FSHarp.Core